Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
@mnewsome I fixed this by calling the view methods `promptNumberOfGuesses()` and `receiveNumberOfGuesses()` from the `main.swift` file, and then passing the returned value into `GuessManager` instance when I create it. Because I only needed `maxNumberOfGuesses` for that one view method, I just set that in the main and passed it to the method, and then removed that variable from the `GuessManager` class